athena/tests/modules/modeling/module_modeling_llm/mock/utils/mock_llm.py

```
@@ -0,0 +1,93 @@
+from pydantic import BaseModel, Field
+from typing import Any, List, Optional, Callable, Type
```

Ruff (0.11.9)

`typing.Callable` imported but unused

Remove unused import: `typing.Callable`

(F401)